examples/cisco/demo_interface.pp

```diff
@@ -36,6 +36,16 @@
       ensure => 'present',
     }
 
+    $ipv4_dhcp_relay_info_trust = platform_get() ? {
+      /(n3k|n7k|n8k|n9k)/ => true,
```

Comment:
Remove the `n8k` reference.  We will likely need to scrub these before the release.